docs/release-instruction.md

```diff
@@ -3,70 +3,132 @@
 ## Pre-release checks
 
 These should run successfully:
- * `make all`;
- * `make test`;
- * `make lint` (should not change any files);
- * `make fmts` (should not change any files);
- * `go mod tidy` (should not change any files);
- * integration tests in [neofs-devenv](https://github.com/nspcc-dev/neofs-devenv).
 
-## Writing changelog
+* `make all`;
+* `make test`;
+* `make lint` (should not change any files);
+* `make fmts` (should not change any files);
+* `go mod tidy` (should not change any files);
+* integration tests in [neofs-devenv](https://github.com/nspcc-dev/neofs-devenv).
 
-Add an entry to the `CHANGELOG.md` following the style established there. Add an
-optional codename, version and release date in the heading. Write a paragraph
-describing the most significant changes done in this release. Add
-`Fixed`, `Added`, `Removed` and `Updated` sections with fixed bug descriptions
-and changes. Describe each change in detail with a reference to GitHub issues if
-possible.
+## Make release commit
 
-Update the supported version of neofs-contract in `README.md` if there were
-changes in releases.
+Use `vX.Y.Z` tag for releases and `vX.Y.Z-rc.N` for release candidates
+following the [semantic versioning](https://semver.org/) standard.
 
-## Tag the release
+Determine the revision number for the release:
 
-Use `vX.Y.Z` tag for releases and `vX.Y.Z-rc.N` for release candidates
-following the [semantic versioning](https://semver.org/) standard. Tag must be
-created from the latest commit of the master branch.
+```shell
+$ export NEOFS_REVISION=X.Y.Z[-rc.N]
```

Comment:
maybe add a script for that then? `export` seems strange in docs for me

Reply:
Thought about script, but most of the job is done manually. I don't see any problem with holding some example commands in guide. They all explained anyway.